apps/blockchain/lib/blockchain/account.ex

```diff
@@ -543,18 +544,20 @@ defmodule Blockchain.Account do
   @spec put_storage(EVM.state(), Address.t(), integer(), integer()) :: EVM.state()
```

Comment:
Does this spec need to be updated to match the new return type?

Reply:
signature didn't change